### PR TITLE
WIP: MGMT-15839: Bump krb5-libs in assisted-installer image

### PR DESCRIPTION
**Pull request by danmanor**
Bump `krb5-libs` in assisted-installer image

### CODE REVIEW

File: Dockerfile.assisted-installer [hunk 1]
```diff
@@ -14,6 +14,8 @@ RUN TARGETPLATFORM=$TARGETPLATFORM make installer
 
 FROM quay.io/centos/centos:stream9
 
+RUN dnf update krb5-libs -y && dnf clean all
```
Comment (osherdp) [inline Dockerfile.assisted-installer:17]: we already have ``1.21.1-1.el9``

**Comment by osherdp** [comment]
@danmanor can we close this?

**Comment by danmanor** [comment]
@osherdp  Thank you for closing this, I missed the notification

**Pull request closed without merging**